modules/host/storageobligations_smoke_test.go

```diff
@@ -494,18 +485,15 @@ func TestMultiSectorStorageObligationStack(t *testing.T) {
 			NewUnlockHash:         types.UnlockConditions{}.UnlockHash(),
 		}},
 	}}
-	err = ht.host.lockStorageObligation(so)
+	ht.host.lockStorageObligation(so.id())
```

Comment:
checking the wrong `err` here, since `h.lockStorageObligation(so.id())` doesn't return an error anymore